hw/top_earlgrey/data/clocks.xdc

```diff
@@ -68,4 +69,9 @@ set_output_delay -clock clk_spi 5 [get_ports SPI_DEV_D1] -add_delay
 create_generated_clock -name clk_spi_in  -divide_by 1 -source [get_ports SPI_DEV_CLK] [get_pins top_*/u_spi_device/u_clk_spi_in_buf/gen_xilinx.u_impl_xilinx/gen_fpga_buf.gen_bufr.bufr_i/O]
 create_generated_clock -name clk_spi_out -divide_by 1 -source [get_ports SPI_DEV_CLK] [get_pins top_*/u_spi_device/u_clk_spi_out_buf/gen_xilinx.u_impl_xilinx/gen_fpga_buf.gen_bufr.bufr_i/O] -invert
 
+## Set asynchronous clock groups
 set_clock_groups -group ${clks_10_unbuf} -group ${clks_48_unbuf} -group ${clks_aon_unbuf} -group clk_io_div2 -group clk_io_div4 -group lc_jtag_tck -group rv_jtag_tck -group {clk_spi clk_spi_in clk_spi_out} -group sys_clk_pin -asynchronous
+
+## The usb calibration handling inside ast is assumed to be async to the outside world
+## even though its interface is also a usb clock.
+set_false_path -from [get_clocks clk_usb_48mhz] -to [get_pins u_ast/u_usb_clk/u_ref_pulse_sync/u_sync*/u_sync_1/gen_*/q_o_reg[0]/D]
```

Comment:
@a-will i finally got around to looking at the vivado results.
At least in my run, the path to the synchronizer inside `ast` has enormous hold time violations (i didn't dig deeper as to why..i assume it has something to do with the BUFG placement).

False pathing the path to the D input however seems to nicely resolve the issue.

Reply:
Ah, I just didn't have enough paths in the report, since they were all going to xbar_main. Incidentally, my build also didn't experience any hold time failures, so yeah... it's probably lucky BUFG placement.

The failing path is actually for clock calibration, so this is fine to be a false path in the FPGA (since we're not actually doing any calibration there).